includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -262,7 +263,7 @@ private function getUpvoteId( $wikiId, $revisionId ) {
 
 		$upvoteId = ( new \WikiaSQL() )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$upvoteId". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)